### ISSUE TITLE
Babel throws error for using deprecated option

### ISSUE THREAD

**Issue by akagomez** [stealjs/steal#772]
Using this list of dependencies:

```
"steal": "git://github.com/stealjs/steal#v1.0.0-pre.7",
"steal-jsx": "git://github.com/stealjs/steal-react-jsx#v0.0.3",
"steal-tools": "git://github.com/stealjs/steal-tools#v1.0.0-pre.1"
```

I get this error in my Steal + React app: 

![](https://i.imgur.com/FjkAJWW.png)

Unless I add `delete options.blacklist` here: https://github.com/stealjs/steal/blob/master/steal.js#L2523

**Comment by matthewp** [comment]
Probably because of this: https://github.com/stealjs/steal-react-jsx/blob/master/package.json#L18-L20  steal-jsx isn't currently compatible with 1.x

**Comment by pYr0x** [comment]
i think we should remove `optional`, `whitelist` and `blacklist` in `options` if we are using babel >= 6?
so we dont get an error by react.
also remove the `babelOptions` in steal-react-jsx and update the readme. Users should use version `0.0.3` for steal < `1.0.0`

**Comment by pYr0x** [comment]
fixed in steal 1.0
